diff --git a/.gitignore b/.gitignore
index 5be420a..1c4ca09 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
 MANIFEST
 README.txt
 dist
+build
diff --git a/gnucashxml.py b/gnucashxml.py
index 2a2599c..e22d171 100644
--- a/gnucashxml.py
+++ b/gnucashxml.py
@@ -407,7 +407,7 @@ def _slots_from_tree(tree):
         value = elt.find(slot + "value")
         type_ = value.get('type', 'string')
         if type_ == 'integer':
-            slots[key] = long(value.text)
+            slots[key] = int(value.text)
         elif type_ == 'numeric':
             slots[key] = _parse_number(value.text)
         elif type_ in ('string', 'guid'):
